.ci/scripts/run_sklearn_tests.py

```diff
@@ -37,16 +37,25 @@
     )
     args = parser.parse_args()
 
-    os.chdir(os.path.dirname(sklearn.__file__))
+    cwd = os.getcwd()
+    sklearn_file_dir = os.path.dirname(sklearn.__file__)
+    os.chdir(sklearn_file_dir)
 
     if os.environ["SELECTED_TESTS"] == "all":
         os.environ["SELECTED_TESTS"] = ""
 
     pytest_args = (
         "--verbose --durations=100 --durations-min=0.01 "
-        f"--rootdir={os.path.dirname(sklearn.__file__)} "
+        f"--rootdir={sklearn_file_dir} "
         f'{os.environ["DESELECTED_TESTS"]} {os.environ["SELECTED_TESTS"]}'.split(" ")
     )
+
+    if rc := os.getenv("COVERAGE_RCFILE"):
+        pytest_args += (
+            f"--cov=onedal --cov=sklearnex --cov-config={rc} "
+            "--cov-report=term".split(" ")

```

Comment:
```suggestion
            "--cov=onedal", "--cov=sklearnex", f"--cov-config={rc}",
            "--cov-report=term",

```

Reply:
removed term as well since its assumed that the reports will be viewed only after integration together.